### DISCUSSION TITLE
Web Based MP3 Download

### DISCUSSION THREAD

**Discussion by thomasjefferson1743** [podverse/podverse-web#1098] [Ideas]
I work out a lot and would like to listen to podcasts on a lightweight MP3 player, a mobile phone is too big, bulky and heavy for wearing while working out. It would be great if I can download podcasts thru the web interface so that I can transfer them to a MP3 player. Lightweight MP3 players are very common for people to workout because mobile phones are simply too heavy. Its way too difficult to transfer a MP3 from a Apple Device to a MP3 player.

Is there a copyright issue, technical issue or development resource issue?

**Comment by mitchdowney** [comment]
hello @thomasjefferson1743! This is a perfectly good feature request, but can you recreate it as a Github Issue instead of a Discussion? A New Issue button can be found here: https://github.com/podverse/podverse-web/issues

If you don't get to it, I will recreate it for you. It's just if you create it then you will continue to get updates as feature work continues.
